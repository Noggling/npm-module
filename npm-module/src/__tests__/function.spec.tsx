import * as React from "react";

import Hello from "..";
import { create } from "react-test-renderer";

test("Component should show 'Blue' text 'Hello Christopher, this is a npm module!'", () => {
  const component = create(<Hello text="Christopher" />);
  const testInstance = component.root;

  expect(testInstance.findByType(Hello).props.text).toBe("Christopher");

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
