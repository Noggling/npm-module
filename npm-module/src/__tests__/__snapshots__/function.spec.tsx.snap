// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'Blue' text 'Hello Christopher, this is a npm module!' 1`] = `
<div
  style={
    Object {
      "color": "blue",
    }
  }
>
  Hello 
  Christopher
  , this is a npm module!
</div>
`;
